### PR TITLE
Changed LOGGING_LEVEL to LOG_LEVEL

### PR DESCRIPTION
**Pull request by IoannisLafiotis**
**What has changed?**

- LOGGING_LEVEL to LOG_LEVEL
- INFLUXDB_LOGGING_LEVEL to INFLUXDB_LOG_LEVEL !!

### CODE REVIEW

File: platform/influxdb.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
         ports:
         - containerPort: 8086
         env:
-        - name: INFLUXDB_LOGGING_LEVEL
+        - name: INFLUXDB_LOG_LEVEL
```
Comment (weberjm) [inline platform/influxdb.yaml]: InfluxDB uses "logging" in its variable declarations

https://docs.influxdata.com/influxdb/v1.8/administration/config/#logging-settings

File: services/reports-analytics/test/ts/env.influxdb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 INFLUXDB_DATA_ENGINE=tsm1
-INFLUXDB_LOGGING_LEVEL=error
+INFLUXDB_LOG_LEVEL=error
```
Comment (weberjm) [inline services/reports-analytics/test/ts/env.influxdb]: InfluxDB uses "logging" for variable declarations.

https://docs.influxdata.com/influxdb/v1.8/administration/config/#logging-settings

**Review by weberjm — CHANGES_REQUESTED** [review]
While we are cleaning up our configuration to use a standard variable for logging, we also must be aware of external system requirements. 

The two commented files which refer to InfluxDB should be reverted to INFLUXDB_LOGGING_LEVEL

File: services/reports-analytics/test/ts/env.influxdb (hunk 2 above)
Comment (IoannisLafiotis) [reply services/reports-analytics/test/ts/env.influxdb]: Changed it back INFLUXDB_LOGGING_LEVEL according to the Docs of influx!

**Review by weberjm — APPROVED** [review]
Thx

**Pull request merged**